js/controller.js

```diff
@@ -259,6 +270,19 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.getVrParams && this.getVrParams()) {
```

Comment:
i think if you use _.isFunction(this.getVrParams)  will be more what you want since, calling this.getVrParams() can still fail if it's string or number.